### PR TITLE
[RUN-10230] added missing flags.

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/Researcher/cli-reference/runai-submit.md [hunk 1]
```diff
@@ -66,12 +66,25 @@ Submit a Job without a name (automatically generates a name)
 runai submit -i gcr.io/run-ai-demo/quickstart -g 1 
 ```
 
+Submit a job without a name to an external URL:
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit.md:69]: What is "a job without a name"?

File: docs/Researcher/cli-reference/runai-submit.md [hunk 2]
```diff
@@ -357,8 +370,17 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 #### -s | --service-type `<string>`
 
-> External access type to interactive jobs. Options are: portforward (deprecated), loadbalancer, nodeport, ingress.
-  
+> External access type to interactive jobs. Options are:
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit.md:373]: @itayvallach do all these options correct?

portforward` (deprecated)
    > * `loadbalancer`
    > * `nodeport`
    > * `ingress`
    > * `external-url

File: docs/Researcher/cli-reference/runai-submit.md [hunk 3]
```diff
@@ -357,8 +370,17 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
...
+    > * `portforward` (deprecated)
```
Comment (liohill) [inline docs/Researcher/cli-reference/runai-submit.md:374]: If it is deprecated, don't we want to omit it?

Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit.md:374]: Usually when something is deprecated, it means you can still use it, but there is a better option which works better. Not always does it mean it does not work. Sometimes there are scripts that use it so we need to let the users know that it is being phased out. Once it is removed we then take it out of the docs and depending upon the effect it has on the users declare it in the release notes.

File: docs/Researcher/cli-reference/runai-submit.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit.md:69]: My assumption was that the user does not want to be "bothered" with making a unique name and relies on the autogenerated unique name. I can change this to:
```suggestion
Submit a job using the system autogenerated name to an external URL:
```

File: docs/Researcher/cli-reference/runai-submit.md (hunk 2 above)
Comment (itayvallach) [reply docs/Researcher/cli-reference/runai-submit.md:373]: ingress doesn't exist anymore. should be deleted. the rest are fine.

Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit.md:373]: @itayvallach Should we remove the deprecated one as well?

Comment (itayvallach) [reply docs/Researcher/cli-reference/runai-submit.md:373]: no I agree with your below comment. it is still functioning.

File: docs/Researcher/cli-reference/runai-submit.md [hunk 4]
```diff
@@ -357,8 +370,17 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
...
+    > * `loadbalancer`
+    > * `nodeport`
+    > * `ingress`
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit.md:377]:
```suggestion
```

**Pull request merged**